projects/documentation/scripts/build-ts.js

```diff
@@ -72,7 +72,13 @@ async function main() {
                 },
             }),
         ],
-        external: ['@spectrum-web-components/*'],
+        external: [
+            '@spectrum-web-components/*',
+            'lit-html',
```

Comment:
Curious to know why the lit packages are added here?

Reply:
Because without it we've been shipping multiple copies to our docs site:

<img width="211" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/1156657/b50ff262-32d3-45ae-ab2e-8448c296338a">